omero/sysadmins/unix/install-web/walkthrough/omeroweb-install-debian-ice3.6.rst

```diff
@@ -1,100 +1,149 @@
-OMERO.web walkthrough installation Debian 9 and IcePy 3.6
-=========================================================
+OMERO.web deployed separately from OMERO.server on Debian 9 and IcePy 3.6
+=========================================================================
 
+Please first read :doc:`../../server-debian9-ice36`.
 
-For convenience in this walkthrough the main OMERO.web configuration options have been defined as environment variables. When following this walkthrough you can either use your own values, or alternatively source the following file::
+
+This is an example walkthrough for installing OMERO.web decoupled from the OMERO.server in a **virtual environment** using OMERO.py and a dedicated system user. Installing OMERO.web in a virtual environment is the preferred way. For convenience in this walkthrough the main OMERO.web configuration options have been defined as environment variables. When following this walkthrough you can either use your own values, or alternatively use the following ones::
     
     OMERO_USER=omero
     WEBPORT=80
     WEBSERVER_NAME=localhost
 
 
 
-Create local user omero, homedir :file:`/home/omero` (run as root)::
-    
-    if [ -z "$(getent passwd omero)" ]; then
-    
-        useradd -m omero
+**The following steps are run as root.**
+
+Create a local system user omero if required, create the homedir too :file:`/home/omero`::
     
-    fi
+    useradd -m omero
     
     chmod a+X /home/omero
 
-Install ZeroC IcePy 3.6. IcePy is managed by PyPI and will be installed as a part of OMERO.web requirements (run as root)::
+Installing prerequisites
+------------------------
+
+**The following steps are run as root.**
+
+Install ZeroC IcePy 3.6. IcePy is managed by `PyPI <https://pypi.python.org/pypi>`_, a package management system used to install and manage software packages written in Python. IcePy and will be installed as a part of OMERO.web requirements::
     
     apt-get -y install libssl-dev libbz2-dev libmcpp-dev libdb++-dev libdb-dev libdb-java
 
-Install other dependencies (run as root)::
+Install other dependencies. The number of dependencies to install depends on the way you plan to install OMERO.web. If you wish to install it in a virtual environment created with ``--system-site-packages`` *on* (**option 1**), you will need to install ``python-pillow`` and ``python-numpy``. If you wish to install it in a virtual environment with ``--system-site-packages`` *off*, few more dependencies will be required (**option 2**)::
     
+    # common dependencies to both options
     apt-get update
     apt-get -y install \
         python-pip \
         python-virtualenv
     
+    apt-get -y install nginx
```

Comment:
Maybe we shouldn't be suggesting `-y`, let them look and check for themselves?

Reply:
Interesting point. It's another difference between the automated docker-testing we do with omero-install and the manual best-practices that we want generated into the docs.

Reply:
That will require some rethinking on the way we have the code snippet included
Probably out the scope of that round

Reply:
When the time comes, it may be as simple as `s/install -y/install/`

Reply:
for omero-install yes not a problem I was more thinking of omeroweb which is another approach
(I need to dig more on that one)

Reply:
alternatively I could go for a simple solution in omeroweb for now
``if doc command without -y else command with -y``
and look at an improved solution later on

Reply:
:+1: Though these docs are already an improvement so fine with me for this issue *not* to block this PR.